src/consts.ts

```diff
@@ -46,7 +46,7 @@ export const DEFAULT_XCM_VERSION = 2;
  * There should only ever be two supported versions.
  * Therefore supported xcm versions should have a fixed length of 2
```

Comment:
Let's fix up the comment here, saying the length. will now always be of 3 since we have 3 versions. Once v5 comes out we can remove v2 etc.